src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -481,8 +481,12 @@ MulticopterAttitudeControl::control_attitude_rates(float dt)
 
 			}
 
+			// I term factor: reduce the gain with increasing rate error (to avoid bounce-back after flips)
```

Comment:
I think it would be nice to have a longer comment here explaining this especially where 400 degrees/seconds comes from.

Reply:
Updated. Is it clear now?

Reply:
Great!